### PR TITLE
renamed base pkg to com.redhat.devtools.intellij.kubernetes (#130)

### LINKED ISSUE #130: rename base package to com.redhat.devtools.intellij.kubernetes
The current base package is **org.jboss.tools.intellij.kubernetes**, it should be renamed to **com.redhat.devtools.intellij.kubernetes**

### PR DESCRIPTION
**Pull request by adietish**
fixes #130

### CODE REVIEW

**Comment by adietish** [comment]
IJ isn't especially good at this. Turned out that it does rename the package(s) but wont refactor the imports.

File: src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java [hunk 1]
```diff
@@ -31,14 +31,14 @@ private KubernetesVersionPriority() {}
      * @return the version with the highest priority
      */
     public static String highestPriority(List<String> versions) {
-        List<Version> byPriority = getByPriority(versions);
+        List<KubernetesVersionFactory.Version> byPriority = getByPriority(versions);
```
Comment (lstocchi) [inline src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java:34]: Here the import is not used anymore or if you want to use the import (which i prefer) it needs to be as before `List<Version>`

File: src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java [hunk 2]
```diff
@@ -31,14 +31,14 @@ private KubernetesVersionPriority() {}
...
         if (byPriority.isEmpty()) {
             return null;
         }
         return byPriority.get(0).getFull();
     }
 
-    private static List<Version> getByPriority(List<String> versions) {
+    private static List<KubernetesVersionFactory.Version> getByPriority(List<String> versions) {
```
Comment (lstocchi) [inline src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java:41]: the same as above

File: src/test/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionFactoryTest.java [hunk 3]
```diff
@@ -37,16 +37,16 @@ public void create_should_create_kubernetes_version() {
     @Test
     public void create_should_create_non_kubernetes_version() {
         // given
-        Version version = KubernetesVersionFactory.create("darthVader");
+        KubernetesVersionFactory.Version version = KubernetesVersionFactory.create("darthVader");
```
Comment (lstocchi) [inline src/test/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionFactoryTest.java:40]: the same as above for all occurrences in this test class.

**Review by lstocchi — APPROVED** [review]
LGTM, just a comment below, maybe IJ changed it for you and you didn't even notice

File: src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java (hunk 2 above)
Comment (adietish) [reply src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java:41]: good catch!

File: src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java (hunk 1 above)
Comment (adietish) [reply src/main/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionPriority.java:34]: good catch!

File: src/test/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionFactoryTest.java (hunk 3 above)
Comment (adietish) [reply src/test/java/com/redhat/devtools/intellij/kubernetes/model/resource/KubernetesVersionFactoryTest.java:40]: good catch!

**Pull request merged**